app/models/wizard.rb

```diff
@@ -1,3 +1,13 @@
 class Wizard < ApplicationRecord
   has_many :products
+  validates :username, presence: true
+  validates :username, uniqueness: true
+  validates :email, presence: true
+  validates :email, uniqueness: true
+  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }
+  
```

Comment:
I'm not sure if Wizard needs an email or will have one but I don't see any problem merging this right now